src/Molecule.py

```diff
@@ -207,21 +228,21 @@ def add_r_group(self):
 
 class FinalMolecule(Molecule):
     """Final molecule with scaffold and two R groups."""
+
     def __init__(self, rgroup1, rgroup2):
         # Name of R groups should be in the form 'Axy' or 'Bxy' e.g. A01 etc.
-        super().__init__()
         self.rgroup1 = rgroup1
         self.rgroup2 = rgroup2
 
     def drug_properties(self):
         """Selects properties of the final drug from the data."""
         drug_properties = [
-                            'pic50',
-                            'clearance_mouse',
-                            'clearance_human',
-                            'logd',
-                            'pampa'
-                            ]
+            'pic50',
+            'clearance_mouse',
+            'clearance_human',
+            'logd',
+            'pampa'
+        ]
```

Comment:
I'm a bit surprised this passed linting in both cases

Reply:
thats actually automatic linting i think! It prefers that to one big line usually